nbqa/__main__.py

```diff
@@ -552,55 +487,59 @@ def _get_configs(
 
     Returns
     -------
-    bool
-        Whether or not to copy __init__.py to temporary directory.
     bool
         Whether to allow nbqa to modify notebooks.
     """
-    config_file, config, config_prefix = _find_config_file(args.command, project_root)
-    nbqa_config = args.nbqa_config
-    allow_mutation = args.nbqa_mutate
+    config_file, config, config_prefix = _find_config_file(
+        cli_args.command, project_root
+    )
+    nbqa_config: Optional[str] = cli_args.nbqa_config
```

Comment:
Is it still necessary to type this, as the types are already set in `CLIArgs`? At least, if I do
```
reveal_type(cli_args.nbqa_config)
```
before this line, then I get
```
$ mypy nbqa/__main__.py 
nbqa/__main__.py:496: note: Revealed type is 'Union[builtins.str, None]'
```

Reply:
Though it seems redundant, I usually write variable declarations with type hints since it explicitly states the type of the declared variable. I find this version more readable (i.e) all type information is present locally. No other reason.

Reply:
OK sure, that's fine with me - if you merge master I think this is ready

Reply:
I will have to pull latest changes in master (ignore_cells changes) and then I will update this PR. After that I will merge.